code/game/antagonist/station/traitor.dm

```diff
@@ -14,7 +14,7 @@
 
 
 /datum/antagonist/traitor/get_extra_panel_options()
-	if(owner.current)
+	if(owner && owner.current)
```

Comment:
It can lost owner?

Reply:
Yes it can. This happens when you manually remove the antag status from someone through the storyteller panel.
The antag datum is left floating without an owner
There's a second button to delete it, but you can also reassign it to a different owner

Either way, this check is necessary to avoid runtime errors when the antag datum is floating in that transient state